craco.config.js

```diff
@@ -0,0 +1,24 @@
+const CracoLessPlugin = require("craco-less");
+
+module.exports = {
+  plugins: [
+    {
+      plugin: CracoLessPlugin,
+      options: {
+        lessLoaderOptions: {
+          lessOptions: {
+            modifyVars: {
+              "@primary-color": "#FDE54E",

```

Comment:
From where you get these values? Have you confirmed this color theme with UI/UX team?

Reply:
From ui,ux doc on drive and double check in figma